### ISSUE TITLE
Add global JS-Properties like "window.x" ?

### ISSUE THREAD

**Issue by nuxodin** [nuxodin/web-namespace-registry#3]
Would it be useful to be able to register for global properties?

**Comment by nuxodin** [comment]
As we have js-modules, global js-properties are no longer recommended anyway?

**Comment by herberthobregon** [comment]
It could be a good idea!
But it could be that it is oriented to create a "namespace"
Something like `window.conectate.x` to avoid possible collisions

**Comment by nuxodin** [comment]
Yes, that's what I mean by global-js namespace, an object that is preferably the only global variable in a framework.

But I don't know if that's needed, you usually don't need global variable at all anymore with JS modules.

Or do you have such a usecase?
